src/Sulu/Bundle/AdminBundle/Resources/js/stores/UserStore/UserStore.js

```
@@ -1,10 +1,123 @@
+        this.loginError = loginError;
+    }
+
+    @action setResetSuccess(resetSuccess: boolean) {
+        this.resetSuccess = resetSuccess;
+    }
+
+    @action setUser(user: User) {
+        this.user = user;
+        moment.locale(user.locale);
+    }
+
+    @action setContact(contact: Contact) {
+        this.contact = contact;
+    }
+
+    @action clearError = () => {
+        this.loginError = false;
+        this.resetSuccess = false;
+    };
+
+    login = (user: string, password: string) => {
+        this.setLoading(true);
+
+        return Requester.post('/admin/v2/login', {username: user, password: password})
+            .then(() => {
+                if (this.user) {
+                    // when the user was logged in already and comes again with the same user
+                    // we don't need to initialize again
+                    if (user === this.user.username) {
+                        this.setLoggedIn(true);
+                        this.setLoading(false);
+
+                        return;
+                    }
+
+                    // the user has now logged in with a different user
+                    // we need to clear the data from the old user
+                    this.clear(true);
+                }
+
+                return initializer.initialize().then(() => {
+                    this.setLoading(false);
+                });
+            })
+            .catch((error) => {
+                this.setLoading(false);
+                if (error.status !== 401) {
+                    return Promise.reject(error);
+                }
+
+                this.setLoginError(true);
+            });
+    };
+
+    resetPassword(user: string) {
+        this.setLoading(true);
+
+        if (this.resetSuccess) {
+            // if email was already sent use different api
+            return Requester.post('/admin/security/reset/email/resend', {user: user})
+                .catch(() => {})
```

Are empty catches a good idea? Shouldn't something happen here?